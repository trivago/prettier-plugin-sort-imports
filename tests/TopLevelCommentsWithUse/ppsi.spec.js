run_spec(__dirname, ['typescript'], {
  importOrderParserPlugins: ['typescript'],
  importOrderSeparation: true,
  importOrderSortSpecifiers: true,
  importOrder: [
    "^@/(.*)$",
    "^[.]"
  ],
});
