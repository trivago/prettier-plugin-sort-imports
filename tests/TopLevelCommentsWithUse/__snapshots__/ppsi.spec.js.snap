// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`top-level-comments.ts - typescript-verify: top-level-comments.ts 1`] = `
// This is a top level comment that should not be duplicated
// This is another comment that should stay at the top

"use server";

import z from 'z';
import c from 'c';
import b from 'b';
import a from 'a';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
// This is a top level comment that should not be duplicated
// This is another comment that should stay at the top

"use server";

import a from "a";
import b from "b";
import c from "c";
import z from "z";
`;
